deploy-service/common/src/main/java/com/pinterest/deployservice/allowlists/BuildAllowlistImpl.java

```diff
@@ -35,4 +37,14 @@ public Boolean trusted(String buildName) {
         }
         return false;
     }
+
+    // sox_compliant checks if build matches trusted URL allow list
+    public Boolean sox_compliant(String buildName) {
+        for (String pattern : soxBuildURLs) {
+            if (buildName.matches(pattern)) {
```

Comment:
can buildNamebe null? same as other places though.

Reply:
It's not nullable in the db schema, thanks for checking